src/azure_extension.cpp

```diff
@@ -35,6 +35,17 @@ static void LoadInternal(DatabaseInstance &instance) {
 	                          "Notice that the result may be incorrect for more than one active DuckDB connection "
 	                          "and the calculation of total received and sent bytes is not yet implemented.",
 	                          LogicalType::BOOLEAN, false);
+	// SPN configuration entries
+	config.AddExtensionOption("azure_tenant_id", "The Microsoft Entra tenant (directory) ID.", LogicalType::VARCHAR);
```

Comment:
You don't need to add the extension options here for new features. We are going to deprecated passing secret information through ExtensionOptions in the future, so any newly added features don't need to support this at all. So if you could just add the `Secret` implement only, that would be amazin

Reply:
Oh that will simplify so much everything! Thx :)
Done in [c459321](https://github.com/duckdb/duckdb_azure/pull/33/commits/c4593214e8d9a070decdc53652679fcdcf9fcf65)